{
  "comments": [
    {
      "key": {
        "uuid": "5ac10518_f6dcd176",
        "filename": "dovetail/report.py",
        "patchSetId": 1
      },
      "lineNbr": 227,
      "author": {
        "id": 5469
      },
      "writtenOn": "2016-11-03T12:13:08Z",
      "side": 1,
      "message": "If db_result is None, line 227 will fail for \u0027KeyError: \u0027criteria\u0027\u0027.",
      "range": {
        "startLine": 227,
        "startChar": 8,
        "endLine": 227,
        "endChar": 56
      },
      "revId": "4647ec2fffaa2628f03c5ee08ee5419cded18315",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}